relay-event-schema/src/protocol/attributes.rs

```diff
@@ -30,6 +30,21 @@ impl Attribute {
             other: Object::new(),
         }
     }
+
+    /// Stores `meta` information in the attribute value, unless it is [`None`].
```

Comment:
I find this docstring a bit confusing. Maybe something like
```suggestion
    /// Converts a `Value` into an `Attribute` while preserving `meta` information.
```